compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -589,6 +603,7 @@ impl<'a> Parser<'a> {
         if ate {
             // See doc comment for `unmatched_angle_bracket_count`.
             self.unmatched_angle_bracket_count += 1;
+            self.angle_args_span_stack.push(self.prev_token.span);
```

Comment:
Can we avoid this span stack and instead just emit a suggestion to insert a `>` instead of suggesting to replace the entire span with something that finishes in a `>`? Or do we need the span for something other than the suggestion?